tests/store/artifact/test_databricks_artifact_repo.py

```diff
@@ -0,0 +1,324 @@
+                as write_credentials_mock, \
+                mock.patch('requests.put') as request_mock:
+            mock_credentials = ArtifactCredentialInfo(signed_uri=MOCK_AWS_SIGNED_URI,
+                                                      type=ArtifactCredentialType.AWS_PRESIGNED_URL,
+                                                      headers=MOCK_HEADERS)
+            write_credentials_response_proto = GetCredentialsForWrite.Response(
+                credentials=mock_credentials)
+            write_credentials_mock.return_value = write_credentials_response_proto
+            request_mock.return_value = mock_response
+            databricks_artifact_repo.log_artifact(test_file.strpath, artifact_path)
+            write_credentials_mock.assert_called_with(MOCK_RUN_ID, expected_location)
+            request_mock.assert_called_with(ANY, ANY,
+                                            headers=expected_headers)
+
+    def test_log_artifact_aws_presigned_url_error(self, databricks_artifact_repo, test_file):
+        with mock.patch(DATABRICKS_ARTIFACT_REPOSITORY + '._get_write_credentials') \
+                as write_credentials_mock, \
+                mock.patch('requests.put') as request_mock:
+            mock_credentials = ArtifactCredentialInfo(signed_uri=MOCK_AWS_SIGNED_URI,
+                                                      type=ArtifactCredentialType.AWS_PRESIGNED_URL)
+            write_credentials_response_proto = GetCredentialsForWrite.Response(
+                credentials=mock_credentials)
+            write_credentials_mock.return_value = write_credentials_response_proto
+            request_mock.side_effect = MlflowException("MOCK ERROR")
+            with pytest.raises(MlflowException):
+                databricks_artifact_repo.log_artifact(test_file.strpath)
+            write_credentials_mock.assert_called_with(MOCK_RUN_ID, ANY)
+
+    @pytest.mark.parametrize("artifact_path", [
+        None,
+        'output/',
+        '',
+    ])
+    def test_log_artifacts(self, databricks_artifact_repo, test_dir, artifact_path):
+        with mock.patch(DATABRICKS_ARTIFACT_REPOSITORY + '.log_artifact') as log_artifact_mock:
+            log_artifact_mock.return_value = None
+            databricks_artifact_repo.log_artifacts(test_dir.strpath, artifact_path)
+            artifact_path = artifact_path or ''
+            calls = [mock.call(os.path.join(test_dir.strpath, 'empty-file'),
+                               os.path.join(artifact_path, '')),
+                     mock.call(os.path.join(test_dir.strpath, 'test.txt'),
+                               os.path.join(artifact_path, '')),
+                     mock.call(os.path.join(test_dir.strpath, 'subdir/test.txt'),
+                               os.path.join(artifact_path, 'subdir'))]
+            log_artifact_mock.assert_has_calls(calls)
+
+    def test_list_artifacts(self, databricks_artifact_repo):
```

Comment:
Can we test that outputs are computed relative to `run_relative_artifact_repo_root_path`?